pkg/liqo-controller-manager/namespaceMap-controller/ensure_remote_namespaces.go

```diff
@@ -126,31 +107,30 @@ func (r *NamespaceMapReconciler) deleteRemoteNamespace(clusterID, remoteName str
 	}
 
 	if remoteNamespace.DeletionTimestamp.IsZero() {
-		if err := r.RemoteClients[clusterID].Delete(context.TODO(), remoteNamespace); err != nil {
+		if err := r.RemoteClients[clusterID].Delete(ctx, remoteNamespace); err != nil {
 			klog.Errorf("unable to delete the namespace '%s' from the remote cluster: '%s'", remoteName, clusterID)
 			return err
 		}
 		klog.Infof("The deletion timestamp is correctly set on the namespace '%s'", remoteName)
 	}
-	return fmt.Errorf("remote Namespace '%s' terminating", remoteName)
+	return fmt.Errorf("the remote Namespace '%s' inside the cluster '%s' is undergoing graceful termination", remoteName, clusterID)
 }
 
 // If DesiredMapping field has less entries than CurrentMapping, is necessary to remove some remote namespaces.
 // This function checks if remote namespaces requested to be deleted are really deleted.
-func (r *NamespaceMapReconciler) ensureRemoteNamespacesDeletion(nm *mapsv1alpha1.NamespaceMap) bool {
+func (r *NamespaceMapReconciler) ensureRemoteNamespacesDeletion(ctx context.Context, nm *mapsv1alpha1.NamespaceMap) bool {
 	errorCondition := false
 	for localName, remoteStatus := range nm.Status.CurrentMapping {
 		if _, ok := nm.Spec.DesiredMapping[localName]; ok && nm.DeletionTimestamp.IsZero() {
 			continue
 		}
-		if err := r.deleteRemoteNamespace(nm.Labels[liqoconst.RemoteClusterID], remoteStatus.RemoteNamespace); err != nil {
+		if err := r.deleteRemoteNamespace(ctx, nm.Labels[liqoconst.RemoteClusterID], remoteStatus.RemoteNamespace); err != nil {
 			nm.Status.CurrentMapping[localName] = mapsv1alpha1.RemoteNamespaceStatus{
 				RemoteNamespace: remoteStatus.RemoteNamespace,
 				Phase:           mapsv1alpha1.MappingTerminating,
 			}
 			errorCondition = true
-			klog.Errorf("%s -> Namespace '%s' cannot be deleted from cluster '%s'", err,
-				localName, nm.Labels[liqoconst.RemoteClusterID])
+			klog.Info(err)
```

Comment:
Why did you reduce the logging and are you using the `klog.Info` instead of `klog.Error`?
I would keep at least the remote clusterID

Reply:
Actually, that log is not even too necessary, because the error is logged inside the function `deleteRemoteNamespace` previously called. I think I will remove that klog, it's just redundant